sdk/identity/identity/review/identity.api.md

```diff
@@ -121,14 +130,21 @@ export const CredentialUnavailableErrorName = "CredentialUnavailableError";
 // @public
 export class DefaultAzureCredential extends ChainedTokenCredential {
     constructor(options?: DefaultAzureCredentialOptions);
+    static readonly credentials: DefaultCredentialConstructor[];
```

Comment:
I wonder if exposing class constructors directly is the best way to do this...

Reply:
This was removed in favor of simply injecting the `keytar` dependency from the extension.